<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainGUI</class>
 <widget class="QMainWindow" name="MainGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EyeExperimentLauncher</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/icons/viewmind.png</normaloff>:/icons/icons/viewmind.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Messages</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="pteLog"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect_to_EyeTracker"/>
   <addaction name="actionCalibrate_Eye_Tracker"/>
   <addaction name="separator"/>
   <addaction name="actionStart_Experiment"/>
   <addaction name="separator"/>
   <addaction name="actionConfigure"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>846</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Eye Tracker</string>
    </property>
    <addaction name="actionConnect_to_EyeTracker"/>
    <addaction name="actionCalibrate_Eye_Tracker"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionConfigure"/>
   </widget>
   <addaction name="menuTools"/>
   <addaction name="menuSettings"/>
  </widget>
  <action name="actionConnect_to_EyeTracker">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/connect.png</normaloff>:/icons/icons/connect.png</iconset>
   </property>
   <property name="text">
    <string>Connect to EyeTracker</string>
   </property>
  </action>
  <action name="actionCalibrate_Eye_Tracker">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/calibrate.png</normaloff>:/icons/icons/calibrate.png</iconset>
   </property>
   <property name="text">
    <string>Calibrate Eye Tracker</string>
   </property>
  </action>
  <action name="actionStart_Experiment">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/experiment.png</normaloff>:/icons/icons/experiment.png</iconset>
   </property>
   <property name="text">
    <string>Start Experiment</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/configuration.png</normaloff>:/icons/icons/configuration.png</iconset>
   </property>
   <property name="text">
    <string>Configure ....</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
